core/modules/qana/ScanTablePlugin.cc

```diff
@@ -182,13 +182,19 @@ ScanTablePlugin::_findScanTables(query::SelectStmt& stmt,
             hasSelectColumnRef = true;
         }
     }
-    // FIXME hasSelectStar is not populated right now. Do we need it?
+
+    auto czar = czar::Czar::getCzar();
+    bool queryDistributionTestVer = czar->getQueryDistributionTestVer();
 
     StringPairVector scanTables;
-    // Right now, queries involving less than a threshold number of
-    // chunks have their scanTables squashed as non-scanning in the
-    // plugin's applyFinal
-    if (hasSelectColumnRef || hasSelectStar) {
+    // Even trivial queries need to use full table scans to avoid crippling the czar
+    // with heaps of high priority, but very simple queries. Unless there are
+    // factors that greatly restrict how many chunks need to be read, it needs to be
+    // a table scan.
+    // For system testing, it is useful to see how the system handles large numbers
+    // trivial queries as the amount of work done by the workers is minimal. This
+    // highlights the cost of czar-worker communications.
+    if (!queryDistributionTestVer || hasSelectColumnRef) {
```

Comment:
What happened to `hasSelectStar`? Will they be now treated as interactive queries?

Reply:
It never did anything. It was just a placeholder (see comment at line 185). Filling in an accurate value for that is not easy. Since 'queryDistributionTestVer' should never be true in a production environment, it shouldn't matter.

Reply:
Thanks! Then it's okay to merge the PR and get it tested.